ftest.sh

```diff
@@ -205,7 +213,11 @@ export D_LOG_FILE=\"$TEST_TAG_DIR/daos.log\"
 mkdir -p ~/.config/avocado/
 cat <<EOF > ~/.config/avocado/avocado.conf
 [datadir.paths]
-logs_dir = $DAOS_BASE/src/tests/ftest/avocado/job-results
+if [ $SL_PREFIX == "/usr" ]; then
```

Comment:
(lint) The double quotes around this do nothing. Remove or escape them. [SC2140]